packages/extension-huggingface/src/index.ts

```diff
@@ -4,7 +4,14 @@ import {
   Runner as HuggingFaceTableQuestionAnsweringFilterRunner,
 } from './lib/filters/tableQuestionAnswering';
 
+import {
+  Builder as TextGenerationFilterBuilder,
+  Runner as TextGenerationFilterRunner,
+} from './lib/filters/textGeneration';
+
 export default [
   HuggingFaceTableQuestionAnsweringFilterBuilder,
   HuggingFaceTableQuestionAnsweringFilterRunner,
+  TextGenerationFilterBuilder,
+  TextGenerationFilterRunner,
```

Comment:
Please ensure that the naming is aligned with HuggingFace, either by using it as a prefix or without it.

Reply:
Thanks for finding the naming issue, it has been fixed at 8c6a02c209ed3fc7af672b641de937c3b42d7a17.